nfs/util/parse.go

```diff
@@ -0,0 +1,31 @@
+package util
```

Comment:
In general I wouldn't advise a package named `util`, but it might be okay for an internal package.  I'd make this an internal package: `procfs/internal/util` or similar.